nautobot/dcim/tables/devicetypes.py

```diff
@@ -157,7 +195,7 @@ class ConsolePortTemplateTable(ComponentTemplateTable):
     actions = ButtonsColumn(
         model=ConsolePortTemplate,
         buttons=("edit", "delete"),
-        return_url_extra="%23tab_consoleports",
+        return_url_extra=r"%3Ftab=consoleports",
```

Comment:
I'm mildly curious why we have to percent-encode the `?` as `%3F` here rather than providing it as a literal `?`.

Reply:
That would construct a url like this `?return_url=/dcim/module-types/86b9d1de-60f3-46fc-b356-eae2ac2cf25f/?tab=modulebays`

Would the second `?` confuse some older or simpler browsers? (seems to work in firefox fwiw)

Reply:
Got it. Makes sense as-is then.